### ISSUE TITLE
Accessibility: Audio Tile missing <track> element

### ISSUE THREAD

**Issue by rnunez80** [castlecms/castle.cms#21]
Ensures <audio> elements have captions

wcag2a
wcag122
section508
section508.22.a

Inspect
Target:#mep_0 > .mejs-inner > .mejs-mediaelement > .pat-audio

HTML:
<audio class="pat-audio" preload="none" style="width: 100%" src="https://demo-cuny.castlecms.io/audio-repository/warningstructuralintegrity_ep.mp3/@@download/file/warningstructuralintegrity_ep.mp3">

Fix all of the following:
The multimedia element does not have a captions track

https://dequeuniversity.com/rules/axe/2.1/audio-caption?application=axeChrome

**Comment by rnunez80** [comment]
One possible solution would be to add a caption field for uploading a ttml or vtt file. if this field exists then the tag is added to all audio or video views
<track kind="captions" src="object_url/caption.vtt" srclang="en-us" label="English" />

**Comment by vangheem** [comment]
We're doing some issue reaping and closing out inactive issues. Most current project management/bug tracking is being done internally at Wildcard Corp. This tracker will remain open for the public to submit and track issues.

If this bug/feature is still relevant and being worked on/needs fixing, please feel free to reopen.
